extension-config.json

```diff
@@ -88,6 +88,60 @@
         "label": "Define a url for the image.",
         "type": "text"
       }
+    },
+    "iosAppVersions": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "",
+      "params": {
+        "label": "Blocked iOS App Versions.",
+        "type": "text"
+      }
+    },
+    "androidAppVersions": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "",
+      "params": {
+        "label": "Blocked Android App Versions.",
+        "type": "text"
```

Comment:
can't this be type json to make this configuration an array so you don't have to work to make it an array later?